Modules/Forum/classes/class.ilObjForumGUI.php

```diff
@@ -97,6 +97,8 @@ class ilObjForumGUI extends \ilObjectGUI implements \ilDesktopItemHandling
     public $rbac;
     public $locator;
     public $ilHelp;
+    /** @var int */
+    public $selectedSorting = ilForumProperties::VIEW_TREE;
```

Comment:
Why did you choose a `public` access modifier here?

Reply:
I had no special reason for this

Reply:
Then please make it `private`.